docs/ref/contrib/gis/functions.txt

```diff
@@ -261,13 +261,16 @@ right-hand rule.
 ``GeoHash``
 ===========
 
-.. class:: GeoHash(expression, **extra)
+.. class:: GeoHash(expression, precision=None, **extra)
 
 *Availability*: PostGIS, SpatiaLite (LWGEOM)
 
 Accepts a single geographic field or expression and returns a `GeoHash`__
 representation of the geometry.
 
+The number of returned characters is controlled by the ``precision`` keyword
```

Comment:
> ("digits" rather than "characters")

Nope, geohash is a really sequence of alphanumeric characters.

Considering your proposal, I'm going to rewrite the sentence as: 
```
The precision keyword argument controls the number of characters in the result.
```
Does that look OK?

> Is it worth mentioning the behavior in the default (None) case?

Description of the default case is pretty long in Postgis docs and MySQL requires `precision` parameter, result differs non-significantly among Postgis and MySQL in some cases, so I think that should be covered by something like this:

```
Consult with backend documentation to learn how exactly precision parameter affects result.
```